Automattic/newspack-listings
# feat: add block patterns

### All Submissions:






### Changes proposed in this Pull Request:



Adds more block patterns for use in listings. Currently only includes only Business Listing patterns as designed by @thomasguillot here: https://newspack-listings-demo.newspackstaging.com/listing-patterns/

Closes #7.

### How to test the changes in this Pull Request:

1. Create a new Place or Marketplace listing.
2. Confirm that all five business listing patterns appear in the block inserter UI and that the layouts and default content match the designs.
3. Confirm that the business listing patterns are not available in any post type other than Place and Marketplace listings.

### Other information:
